Main.elm

```diff
@@ -28,15 +28,45 @@ init flags =
         authorLimit =
             flags.authorLimit
 
+        showInstitution =
```

Comment:
I don't see the point of declaring these variable (I would just write `showInstitution = flags.showInstitution` on line 67 etc) but if you do want to get the variables, you can destructure the flags record: `{showInstitution, showCity, ... } = flags` instead